### ISSUE TITLE
Crash for shedulled livestream

### ISSUE THREAD

**Issue by georgkrause** [mister-monster/YouTube2PeerTube#17]
When there is a sheduled live-stream, the video obviously cannot be downloaded which leads to following crash:

```
Traceback (most recent call last):
  File "youtube2peertube.py", line 324, in <module>
    run(run_once=False)
  File "youtube2peertube.py", line 320, in run
    run_steps(conf)
  File "youtube2peertube.py", line 296, in run_steps
    pt_result = upload_to_pt(dl_dir, channel_conf, queue_item, access_token, thumb_extension)
  File "youtube2peertube.py", line 180, in upload_to_pt
    ("videofile", get_file(video_file)),
  File "youtube2peertube.py", line 146, in get_file
    return (path.basename(file_path), open(path.abspath(file_path), 'rb'),
FileNotFoundError: [Errno 2] No such file or directory: '*********'
```

**Comment by mister-monster** [comment]
Thanks for reporting this. I have to find a way to tell if it is a live stream and ignore it until the video is complete.

**Comment by mister-monster** [comment]
Closed, commit number fbc2856 fixed this issue
